recipe/run_conda_forge_build_setup_osx

```diff
@@ -7,20 +7,22 @@ export CPU_COUNT=2
 export PYTHONUNBUFFERED=1
 
 # TODO: this has moved to conda_build_config.yaml, but we are keep it here for a little while, until
-#   the dust settles on using conda-build 3.  If this value differs from the one in conda_build_config.yaml,
-#   things might get confusing, so make sure to keep them the same, and remove this value when the new
-#   way is working well enough.
-export MACOSX_DEPLOYMENT_TARGET="10.9"
-
-if [[ "$CIRCLECI" == "true" || "$OSX_FORCE_SDK_DOWNLOAD" == "1" ]]; then
-    echo "Downloading sdk"
-    curl -L -O https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.9.sdk.tar.xz
-    tar -xf MacOSX10.9.sdk.tar.xz -C $(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/
-    sed -i '' "s|<string>10.11</string>|<string>10.9</string>|g" $(xcode-select -p)/Platforms/MacOSX.platform/Info.plist
-fi
+#   the dust settles on using conda-build 3.
+#   This now only sets the value when it is undefined,
+#   which will eventually become entirely redundant.
+#   Remove this value when the new way is working well enough.
+export MACOSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET:-10.9}
```

Comment:
Sorry I think I'm missing something, but what is the new way? Would it be possible to link to relevant docs, issue(s), or PR(s) in the comment to provide additional information? The fewer links and the closer to proper docs the better, but I understand if that isn't totally possible.

Reply:
I just rephrased the prior comment, so I'm not sure. I think MACOSX_DEPLOYMENT_TARGET is set in conda_build_config.yaml by conda-smithy now that we are using cb3, so this should expect to be defined, which would make this line do nothing.

Reply:
Updated the comment. It's set [in conda-forge-pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/5efc55669c864b57bc9bb8b6557ee9175310d681/recipe/conda_build_config.yaml#L51) and can be overridden in recipes if need be. Setting the default here should only take effect for feedstocks still on conda-build 2.

Reply:
Right, ok, conda-build 2 compatibility, that makes sense. Sorry wasn't thinking about that. Thanks for clarifying.